DEV_ENV_SETUP.md

```diff
@@ -0,0 +1,69 @@
+# Developer Environment Setup Guide for niceML 🍦
+
+## Overview
+
+👋 Welcome to the development environment setup guide for niceML! This guide will walk you through the process of setting up a development environment for contributing to our open-source project. We use Poetry for managing dependencies, and we recommend using conda environments for a seamless development experience. Let's get your environment up and running smoothly! 🚀
+
+## Prerequisites
+
+Before you begin, ensure that you have the following installed on your system:
+
+- [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- [Poetry](https://python-poetry.org/docs/#installation)
+- [Conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html) (miniconda or anaconda)
```

Comment:
I think pipenv is also allowed, isn't it?

Reply:
Add a hint for pipenv or other solutions. But I think we should recommend our preferred setup (conda)